power_grid_model_io_native_c/power_grid_model_io_native/include/power_grid_model_io_native/vnf_converter/vnf_pgm_converter.hpp

```diff
@@ -86,8 +114,19 @@ inline char* PgmVnfConverter::get_file_buffer() { return this->f_file_buffer; }
 
 inline power_grid_model::WritableDataset* PgmVnfConverter::get_deserialized_data() { return this->deserialized_data; }
 
-inline void PgmVnfConverter::convert_node_input() {
-    // Implementation
+inline std::string const& PgmVnfConverter::get_serialized_data() const { return this->serialized_data; }
+
+inline std::vector<power_grid_model::Node>
+PgmVnfConverter::convert_node_input() { // NOLINT(readability-convert-member-functions-to-static)
```

Comment:
please either follow the suggestion (make it `static`) or use member variables. it's a design choice i leave up to you and we can change it at any later time cause it doesn't affect the API, but disabling the warning doesn't sound like a good plan 😄